source/common/stats/isolated_store_impl.h

```diff
@@ -81,11 +79,12 @@ class IsolatedStoreImpl : public Store {
     return std::vector<ParentHistogramSharedPtr>{};
   }
 
+  void clear();
```

Comment:
Does this only get called in tests? If so, rename it something test-appropriate (or use a test peer? not sure what best practice is). Else, please provide docs -- does this get called at shutdown?

Reply:
I definitely don't need it in this PR and I'm not sure I really need it in future ones either. Removed.